docs/api/tutorials/subscriptions.md

```diff
@@ -0,0 +1,108 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Subscriptions
+
+<FeatureAvailability saasOnly />
+
+## Why Would You Use Subscriptions on Datasets?
+
+Subscriptions are a way to receive notifications when entity changes occur (e.g. deprecations, schema changes, ownership changes, etc.) or when assertions change state (pass, fail, or error). Subscriptions can be created at the dataset level (affecting any changes on the dataset, as well as all assertions on the dataset) or at the assertion level (affecting only specific assertions).
+
+### Goal Of This Guide
+
+This guide specifically covers how to use the [DataHub Cloud Python SDK](https://pypi.org/project/acryl-datahub-cloud/) for managing Subscriptions:
+
+- Create: create a subscription to a dataset or assertion.
+- Remove: remove a subscription.
+
```

Comment:
Does managing subscriptions require some privilege? if so, the corresponding `Prerequisites` section should be added here.